README.md

```diff
@@ -10,7 +10,46 @@ First, you'll need to configure a database. Once this project is deployed, we'll
+3. In your powershell (admin), run `choco install postgresql10` (powershell (admin) can be found by pressing `win + x`). 
+- Make sure you use the admin powershell throughout all the steps.
+4. Upgrade postgresql: `choco upgrade postgresql`.
+5. Add `C:\Program Files\PostgreSQL\10\bin` to your [paths](https://helpdeskgeek.com/windows-10/add-windows-path-environment-variable/).
+6. Restart your admin terminal to update the path. 
+7. Start postgres: `psql -U postgres` you will see postgres-# in the terminal.
+8. Create your user with `CREATE USER name;` The terminal will print CREATE ROLE.
+9. Alter your role with permission to create a database with `ALTER USER name WITH CREATEDB CREATEROLE;` The terminal will print ALTER ROLE
+10. Check everything is in order with `\du`. The terminal will print a table with Role name, Atrributes, and Member of columns. If your user is listed you are good to go.
+11. Quit with `\q` or `ctr + c`.
+12. Start psql with new username with `psql`. You should see `name=>`.
+13. Create a database with: `CREATE DATABASE resources OWNER name;`.
+14. Connect to the resources database: `\c resources`. If you see resources => you are ready to move on to the next steps.
+## Set Up and Activate your Environment:
+15. Set the connection information in a [user environment variable](https://docs.microsoft.com/en-us/windows/desktop/shell/user-environment-variables):
+- Search --> Edit the System Environment Variables --> Environment Variables... (at bottom) --> System Variables: New... --> 
+    - Variable Name: `SQLALCHEMY_DATABASE_URI`
+    - Variable Value: `postgresql://aaron:password@127.0.0.1:5432/resources` (use your chosen username, password and database name)
+16. In powershell navigate to where you would like to clone a fork of your repository. `cd \path\to\directory`
+16. Clone your fork of the project `git clone {project_url} resources`
+17. Change directory into the project `cd resources-api`
+18. [Create a virtual environment](https://docs.python.org/3/library/venv.html) called venv `python -m virtualenv venv`.
+19. Set Execution Policy to unrestricted `Set-ExecutionPolicy Unrestricted -Force`
+20. Activate virtual environment venv\Scripts\Activate.ps1
+21. Install the required dependencies `pip install -r requirements.txt`
+22. Set the FLASK_APP environment variable (see step 15) 
+- Variable name: `FLASK_APP`
+- Variable Value: `run.py`
+- Optionally, enable debugging by setting the environment to development with the Variable Value: `development`
+24. Restart powershell.
+23. Create the tables in your database with `flask db_migrate create_tables`
+24. Tell flask that your database is up to date with `flask db stamp head`
+25. Populate your database with the resources `flask db_migrate init`
+26. Start your development server with `flask run` and you're ready to go!
+
+27. Check your work: `Ctrl + c` then `psql resources`. The terminal pointer will say `resources=>`. Then `\dt` to view the schema.
```

Comment:
I think perhaps the check should be to navigate to `localhost:5000/api/v1/resources` and see a bunch of resources, huh? If you don't, then you know _something_ went wrong. Not everyone will know what they're looking for when they see the schema, and it doesn't prove that everything made it into those tables just because the schema is correct.